OpenSim/Examples/ExampleHopperDevice/buildTestbedModel.cpp

```diff
@@ -1,7 +1,5 @@
-#ifndef _helperMethods_h_
-#define _helperMethods_h_
 /* -------------------------------------------------------------------------- *
- *                         OpenSim:  helperMethods.h                          *
+ *                   OpenSim:  buildTestbedModel.cpp                          *
```

Comment:
Was there a reason why it was a header file as opposed to a cpp file like the others?